api/Event.json

```diff
@@ -1042,6 +1042,54 @@
           }
         }
       },
+      "search": {
```

Comment:
Actually, on reading the MDN page, I realize I've misunderstood what this feature is for. I think this ought to be renamed `search_event` and be moved to [`HTMLInputElement.json`](https://github.com/mdn/browser-compat-data/blob/master/api/HTMLInputElement.json) instead. Sorry for the confusion and thank you!

Reply:
After checking, I made a mistake on the file to modify to add the `search_event` , but are you sure that it should be in `HTMLInputElement.json` ? Shouldn't it be in `GlobalEventHandlers.json` just like `oninput` ? (And I should rename it `onsearch`) Thank you in advance!

Reply:
I think we can have both, actually. An `onsearch` feature in `GlobalEventHandlers.json` (for the property that you assign a handler to) and a `search_event` feature for the event itself, as a subfeature of the applicable type.

That said, how we represent events has changed a bit recently (which is partly what caused my initial confusion), so maybe @Elchi3 or @wbamberg can confirm or correct this.

Reply:
Thanks for testing and providing the fiddle!

Yes, we're representing event BCD separately for the `addEventListener('eventName', ...)` and the `oneventName` variants, since it's possible that the BCD differs.

So we could have two entries:

* one named `search_event`, with a field like `"description": "<code>search</code> event"`, under `HTMLInputElement`. That's needed because if we just call it `search` then it might clash with a method or property name. See https://github.com/mdn/browser-compat-data/blob/master/api/HTMLElement.json#L146-L194 for an example.

* one named `onsearch` with no `"description"` field. See https://github.com/mdn/browser-compat-data/blob/master/api/GlobalEventHandlers.json#L104-L154 for an example. I'm not sure where this one should go (since there's no spec to tell us). Maybe it should also live in `HTMLInputElement`?

BTW, this event will get a bit of extra attention in https://github.com/mdn/sprints/issues/1126, so this is a very timely PR.

Reply:
Thank you for taking the time to enlighten me, especially for my first contribution! I will modify my PR accordingly.